libzim/writer.py

```diff
@@ -135,7 +135,7 @@ class Creator:
         Zim file metadata
     """
 
-    def __init__(self, filename, main_page, index_language, min_chunk_size):
+    def __init__(self, filename, main_page="", index_language="eng", min_chunk_size=2048):
```

Comment:
I would not provide a default value for the main_page.
We need a value and there is no other way to set it than at `__init__` step.

Reply:
OK, so the default in the wrapper is just for technical reason then. I get it.